src/BaGet.Core/BaGet.Core.csproj

```diff
@@ -15,6 +15,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\BaGet.Protocol\BaGet.Protocol.csproj" />
+    <PackageReference Include="Google.Cloud.Storage.V1" Version="1.0.0" />
```

Comment:
Please create a new project `BaGet.Gcp.csproj` and move `GoogleBucketStorageOptions` and `GoogleBucketStorageService` there

Reply:
See https://github.com/loic-sharma/BaGet/pull/144 as an example